src/footprintmonitoring/azext_footprintmonitoring/tests/latest/test_footprintmonitoring_scenario.py

```diff
@@ -0,0 +1,297 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+import os
+from azure.cli.testsdk import ScenarioTest
+from .. import try_manual, raise_if
+from azure.cli.testsdk import ResourceGroupPreparer
+
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+
+@try_manual
+def setup(test, rg):
+    pass
+
+
+# EXAMPLE: /profiles/put/profilesCreateOrUpdate
+@try_manual
+def step__profiles_put_profilescreateorupdate(test, rg):
+    test.cmd('az footprintmonitoring profile create '
+             '--location "westus2" '
+             '--measurement-count 3 '
+             '--start-delay-milliseconds 5000 '
+             '--tags key1="value1" key2="value2" '
+             '--name "{myProfile}" '
+             '--resource-group "{rg}"',
+             checks=[])
+
+
+# EXAMPLE: /profiles/get/List Footprint profiles in a Resource Group
+@try_manual
+def step__profiles_get_list_footprint_profiles_in_a_resource_group(test, rg):
+    test.cmd('az footprintmonitoring profile list '
+             '--resource-group "{rg}"',
+             checks=[])
+
+
+# EXAMPLE: /profiles/get/List Footprint profiles in a Subscription
+@try_manual
+def step__profiles_get_list_footprint_profiles_in_a_subscription(test, rg):
+    test.cmd('az footprintmonitoring profile list '
+             '-g ""',
+             checks=[])
+
+
+# EXAMPLE: /profiles/get/profilesGet
+@try_manual
+def step__profiles_get_profilesget(test, rg):
+    test.cmd('az footprintmonitoring profile show '
+             '--name "{myProfile}" '
+             '--resource-group "{rg}"',
+             checks=[])
+
+
+# EXAMPLE: /profiles/patch/profilesUpdate
+@try_manual
+def step__profiles_patch_profilesupdate(test, rg):
+    test.cmd('az footprintmonitoring profile update '
+             '--tags key1="value1" key2="value2" '
+             '--name "{myProfile}" '
+             '--resource-group "{rg}"',
+             checks=[])
+
+
+# EXAMPLE: /experiments/put/experimentsCreateOrUpdate
+@try_manual
+def step__experiments_put_experimentscreateorupdate(test, rg):
+    test.cmd('az footprintmonitoring experiment create '
+             '--name "{myExperiment}" '
+             '--description "An experiment description." '
+             '--profile-name "{myProfile}" '
+             '--resource-group "{rg}"',
+             checks=[])
+
+
+# EXAMPLE: /experiments/put/experimentsCreateOrUpdate
+@try_manual
+def step__experiments_put_experimentscreateorupdate(test, rg):
+    test.cmd('az footprintmonitoring experiment create '
```

Comment:
Should it be update test? @qiaozha

Reply:
@changlong-liu have we resolved the issue of this?  looks like appearing again?

Reply:
need to be fix in az